Animation.lua

```diff
@@ -195,6 +197,8 @@ Animation.new = function (args)
     -- Glib.Timer's reference
     self.timer = nil
 
+    self.timer_running = false
```

Comment:
I think timer_running doesn't have much sense as a property. The simple fact that timer exists already tells whether it's running.

Reply:
I don't think that assumption holds true in the current incarnation of your `master` branch, because it doesn't look like the `timer` property is getting set to nil when animations complete normally:

https://github.com/Aire-One/awesome-AnimationFramework/blob/f2b7bba792391e5fc739952e24641a970f7a1755/Animation.lua#L217-L220

I assumed there was some sort of justification for this apparent difference, so I took care to preserve it. If that's not necessary, do I have your permission to update the timer function so that the side effects it has on the `timer` and `tween` properties are identical to that of the `Animation.stopAnimation` method? Doing that should make it possible to simply check `timer` in our condition.